freeplane/src/main/java/org/freeplane/core/ui/image/BigBufferedImage.java

```diff
@@ -192,6 +192,18 @@ public void run() {
 			}.start();
 		}
 
+		/**
+		 * There appears to be a bug in the Java code associated with MappedByteBuffer instances (maybe other
+		 * related classes as well?) in that closing them generates a FileNotFoundException.  For details about
+		 * attempts to get around this problem, see the following  
```

Comment:
Our problem is rather that file.delete() is ignored and the files can not be deleted even on exit.